src/Measure.ts

```diff
@@ -20,7 +21,7 @@ export interface Group {
   rateAggregation?:string;
   improvementNotation?:string;
   scoringUnit?:string;
-  measureGroupTypes?: string[];
+  measureGroupTypes: MeasureGroupTypes[];
```

Comment:
not part of your task. Just don't recollect why we named this measureGroupTypes instead of measureTypes

Reply:
In MAT it is measureType but since in Madie it is moved to Group level, I guess it's renamed.

Reply:
makes sense. thanks!